src/vuex/modules/tx.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
Where is this used? I'm probably being blind here

Reply:
that was a demo for all the tx types. somehow ended up there. I removed it.